ts/editor/FieldState.svelte

```diff
@@ -11,9 +11,6 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         display: flex;
         justify-content: flex;
         flex-grow: 1;
-
-        & > :global(*) {
-            margin-left: 2px;
-        }
+        gap: 6px;
```

Comment:
Unfortunately not - the minimum target is currently Chromium 77 (for the Qt5 Mac build), and iOS 13.4. Bumping iOS to 14.5 may be possible in a few months when iOS 16 comes out, but even if we switched the Mac build to Qt 5.15, it would still not be enough for this, as 5.15 is Chromium 83: https://caniuse.com/?search=flex-gap

Reply:
@kleinerpirat This is still needs to be adressed unfortunately. In the past I've solved it by using `... > :global(*) { margin-left: npx }`

Reply:
Forgot about that one, thanks :v: